keyboards/zeix/eden/info.json

```diff
@@ -0,0 +1,946 @@
+{
+    "keyboard_name": "Eden",
+    "maintainer": "itsme-zeix",
+    "manufacturer": "dnworks",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4C27",
+        "pid": "0x2901",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "GP29",
+        "on_state": 1
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP6","GP5","GP4","GP0","GP12"],
+        "cols": ["GP28","GP27","GP26","GP25","GP24","GP23","GP22","GP3","GP2","GP1","GP8","GP9","GP10","GP11","GP7"]
```

Comment:
```suggestion
        "rows": ["GP6", "GP5", "GP4", "GP0", "GP12"],
        "cols": ["GP28", "GP27", "GP26", "GP25", "GP24", "GP23", "GP22", "GP3", "GP2", "GP1", "GP8", "GP9", "GP10", "GP11", "GP7"]
```

Reply:
Applied changes as suggested. Thank you for the review!